Mathlib/CategoryTheory/Galois/Basic.lean

```diff
@@ -202,14 +202,37 @@ noncomputable def fiberEqualizerEquiv {X Y : C} (f g : X ⟶ Y) :
   · exact Types.equalizerIso (F.map f) (F.map g)
 
 /-- The fiber of the pullback is the fiber product of the fibers. -/
-@[simp]
 noncomputable def fiberPullbackEquiv {X A B : C} (f : A ⟶ X) (g : B ⟶ X) :
     F.obj (pullback f g) ≃ { p : F.obj A × F.obj B // F.map f p.1 = F.map g p.2 } := by
   apply Iso.toEquiv
   apply Iso.trans
   · exact PreservesPullback.iso (F ⋙ FintypeCat.incl) f g
   · exact Types.pullbackIsoPullback (F.map f) (F.map g)
 
+/-- The fiber of the binary product is the binary product of the fibers. -/
+noncomputable def fiberBinaryProductEquiv (X Y : C) :
+    F.obj (X ⨯ Y) ≃ F.obj X × F.obj Y := by
+  apply Iso.toEquiv
+  apply Iso.trans
+  · exact PreservesLimitPair.iso (F ⋙ FintypeCat.incl) X Y
+  · exact Types.binaryProductIso (F.obj X) (F.obj Y)
```

Comment:
```suggestion
noncomputable def fiberBinaryProductEquiv (X Y : C) :
    F.obj (X ⨯ Y) ≃ F.obj X × F.obj Y :=
  (PreservesLimitPair.iso (F ⋙ FintypeCat.incl) X Y ≪≫
    Types.binaryProductIso (F.obj X) (F.obj Y)).toEquiv
```
(I think that `fiberEqualizerEquiv` and `fiberPullbackEquiv` should be modified similarly.)